### PR TITLE
Add a Test Sample for the Drakvuf Feature Extractor

### PR DESCRIPTION
**Pull request by yelhamer**
This PR adds a limited sample report for the drakvuf sandbox unit tests. These tests can be found in PR (https://github.com/mandiant/capa/pull/2143)

### CODE REVIEW

**Comment by williballenthin** [comment]
would you PM me a complete file so I can play around with the performance aspect, too?

**Comment by yelhamer** [comment]
> would you PM me a complete file so I can play around with the performance aspect, too?

Apologies for the late reply. I have sent the sample report your way :)

File: 3da7c2c70a2d93ac4643f20339d5c7d61388bddd77a4a5fd732311efad78e535.elf_
Comment (mr-tz) [inline 3da7c2c70a2d93ac4643f20339d5c7d61388bddd77a4a5fd732311efad78e535.elf_:1]: what is this elf file here?

Comment (yelhamer) [reply 3da7c2c70a2d93ac4643f20339d5c7d61388bddd77a4a5fd732311efad78e535.elf_:1]: It's a test file that got added to master after I had opened this PR. When I merged master into this PR branch the commits in question got duplicated for some reason (perhaps wrong merge strategy?).

Comment (mr-tz) [reply 3da7c2c70a2d93ac4643f20339d5c7d61388bddd77a4a5fd732311efad78e535.elf_:1]: ah, can you rebase to master?

**Pull request merged**